docs/02-procedures/02-learning-goals.adoc

```diff
@@ -62,19 +62,21 @@
 * Abgrenzung und Zusammenhänge zwischen Referenzmodellen auf der Unternehmens- und der Softwarearchitektur-Ebene kennen.
 
 [[LZ-2-8]]
-==== LZ 2-8: EAM-Reifegrad:
-* Beispiel eines Unternehmensarchitektur-Reifegradmodelle kennen, z.B.:
-** CMMI
-** EAMMF der GAO
-**  OMB EA Assessment Framework
-** Gartner EA Maturity Assessment Framework
-* Zusammenhänge zwischen dem EAM-Reifegrad und den Handlungsmöglichkeiten verstehen.
+==== LZ 2-8: EAM-Repository
+* Funktion und Nutzen von EAM-Repositories verstehen, z.B. Historisierung, Deltaermittlung und Versionierung.
+* Herausforderungen bei der Pflege und die Aktualisierung der Daten in der Repository kennen.
```

Comment:
Grammatikfehler: Bitte korrigieren zu:
Herausforderungen bei der Pflege und Aktualisierung der Daten in das Repository kennen.

Reply:
Ich habe den Text zu "in das Repository" geändert.